conf/prod.routes

```diff
@@ -3,4 +3,4 @@
 ->         /                            health.Routes
 ->         /template                    template.Routes
 
-GET        /admin/metrics               com.kenshoo.play.metrics.MetricsController.metrics
+GET        /admin/metrics               @com.kenshoo.play.metrics.MetricsController.metrics
```

Comment:
Is this lib still using static rather than DI injected routing ?

Reply:
Both this file and the original app.routes file still have @ in them so I have removed them.